### ISSUE TITLE
Code fails when no affinity matrix and gamma = 0

### ISSUE THREAD

**Issue by jolaem** [alan-turing-institute/affinity-vae#65]
Not all tensors in the final loss function are on the same device.

Solution:
``affin_loss = torch.Tensor([0]).to(self.device)`` line 180 in ``data.py``.
